corehq/apps/linked_domain/tests/test_update_other_fixtures.py

```diff
@@ -0,0 +1,107 @@
+from django.test import TestCase
+from corehq.apps.fixtures.dbaccessors import (
+    get_fixture_data_types,
+    get_fixture_data_type_by_tag,
+    get_fixture_items_for_data_type,
+    delete_fixture_items_for_data_type
+)
+from corehq.apps.fixtures.models import (
+    FixtureDataType, FixtureTypeField, FixtureDataItem, FieldList, FixtureItemField
+)
+from corehq.apps.linked_domain.exceptions import UnsupportedActionError
+
+from corehq.apps.linked_domain.models import DomainLink
+from corehq.apps.linked_domain.updates import update_fixture
+
+
+def delete_all_domain_fixtures(domain):
+    for fixture in get_fixture_data_types(domain):
+        delete_fixture(fixture)
+
+
+def delete_fixture(fixture):
+    delete_fixture_items_for_data_type(fixture.domain, fixture._id)
+    fixture.delete()
+
+
+def items_to_rows(items):
+    if not items:
+        return []
+
+    # This makes the assumption that all items share the same keys, which are defined in the first item
+    header = [header_name for header_name in items[0].fields.keys()]
+    rows = [[val.field_list[0].field_value for val in item.fields.values()] for item in items]
+    return (header, rows)
+
+
+class FixtureUpdateTests(TestCase):
+    def setUp(self):
+        self.upstream_domain = 'upstream-domain'
+        self.downstream_domain = 'downstream-domain'
+        self.link = DomainLink(linked_domain=self.downstream_domain, master_domain=self.upstream_domain)
+        super().setUp()
+
+    def tearDown(self):
+        delete_all_domain_fixtures(self.upstream_domain)
+        delete_all_domain_fixtures(self.downstream_domain)
```

Comment:
LookupTables are stored in Postgres, so they should be cleaned up automatically on test transaction rollback. Does this do something other than deleting SQL data that was created by the test?

Reply:
Nope, this entire file appears to not have been updated since the SQL changes. Removed this section